open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb

```diff
@@ -0,0 +1,1724 @@
+{
```

Comment:
<p>Can this model be downloaded locally and uploaded to cos? Try to do so if it is possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>